web-app/js/portal/cart/PythonDownloadHandler.js

```diff
@@ -37,7 +37,7 @@ Portal.cart.PythonDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
         var _this = this;
 
         return function(collection) {
-            return collection.getLayerState().getSelectedLayer().getFeatureRequestUrl(
+            return OpenLayers.Layer.WMS.prototype.getFeatureRequestUrl(
```

Comment:
I'd prefer to leave it in this file, rather than introduce a `utils` class, as then tend to become dumping grounds for lots of unrelated stuff. 

The namespace still makes sense also, so I'd like to keep that the same.

Maybe I can just drop the `prototype`?